docs/source/osds/scenarios.rst

```diff
@@ -187,10 +187,19 @@ is only available when the ceph release is Luminous or newer.
    The creation of the logical volumes is not supported by ``ceph-ansible``, ``ceph-volume``
    only creates OSDs from existing logical volumes.
 
-``lvm_volumes`` is the config option that needs to be defined to configure the
-mappings for devices to be deployed. It is a list of dictionaries which expects
-a volume name and a volume group for logical volumes, but can also accept
-a partition in the case of ``filestore`` for the ``journal``.
+``lvm_volumes`` or ``devices`` are the config option that needs to be defined to deploy OSDs
```

Comment:
I think this needs a sub-section, one for each of the config options:

    Configurations
    ^^^^^^^^^^^^^^^
    ....

    - ``lvm_volumes`` is a list of [...]

    - ``devices`` is a list of raw device names [...]

That way the separation is clear, and there is no need to have to read this sentence to grasp that there is more than one way to configure this